lib/collection/src/shards/shard.rs

```diff
@@ -151,7 +151,10 @@ impl Shard {
     pub async fn shard_recovery_point(&self) -> CollectionResult<RecoveryPoint> {
         match self {
             Self::Local(local_shard) => Ok(local_shard.recovery_point().await),
+
+            // TODO: Is it good idea to return `RecoveryPoint` for `ForwardProxyShard`? 🤔
```

Comment:
Maybe not, but my thought was that it's still a regular shard internally.

I guess we can assume how that we only request a recovery point if we want to recover this node, in which case it should not have a forward proxy.

So I'm fine by limiting it to be cautious, and removing this here.